pkg/scheduler/framework/runtime/framework.go

```diff
@@ -643,14 +658,15 @@ func (f *frameworkImpl) RunPreFilterExtensionAddPod(
 	podInfoToAdd *framework.PodInfo,
 	nodeInfo *framework.NodeInfo,
 ) (status *framework.Status) {
+	logger := klog.FromContext(ctx)
 	for _, pl := range f.preFilterPlugins {
 		if pl.PreFilterExtensions() == nil {
 			continue
 		}
 		status = f.runPreFilterExtensionAddPod(ctx, pl, state, podToSchedule, podInfoToAdd, nodeInfo)
 		if !status.IsSuccess() {
 			err := status.AsError()
-			klog.ErrorS(err, "Failed running AddPod on PreFilter plugin", "plugin", pl.Name(), "pod", klog.KObj(podToSchedule))
+			logger.Error(err, "Plugin failed")
```

Comment:
Are really plugin, pod and possibly `AddPod` and `PreFilter` strings already in the logger context?

Reply:
Now they are, but still contextualized logging will be disabled by default while it is not GA, so added these keys where it was before, just changed them to be actually keys and not words inside the message. We need to remove these duplicate keys when  contextualized logging hits GA